mpi4jax/cython/__init__.py

```diff
@@ -1,5 +1,13 @@
 from . import mpi_xla_bridge
 from jax.lib import xla_client
 
+import os
+
+mpi_xla_bridge.set_logging(
+    os.environ.get("MPI4JAX_DEBUG", "").lower() in ("true", "1", "on")
+)
+
```

Comment:
we should document in the readme that ENV variable

Reply:
Done